api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -639,6 +650,32 @@ def test_github_webhook_delete_installation(
     assert not GithubConfiguration.objects.filter(installation_id=1234567).exists()
 
 
+def test_github_webhook_with_non_existing_installation(
+    github_configuration: GithubConfiguration,
```

Comment:
nit:
```suggestion
    github_configuration: GitHubConfiguration,
```

Reply:
I think this shouldn't be changed as it's the model name.